translate/storage/txt.py

```diff
@@ -145,14 +145,8 @@ def parse(self, lines):
             unit = self.addsourceunit("\n".join(block))
             unit.addlocation("%s:%d" % (self.filename, startline + 1))
 
-    def serialize(self):
-        source = self.getoutput()
-        if isinstance(source, six.text_type):
-            return source.encode(self.encoding)
-        return source
-
-    def getoutput(self):
```

Comment:
I agree with normalizing the API, but this wasn't a private method, so I am not sure if just removing it is the best approach.